.env

```diff
@@ -1,6 +1,6 @@
-REST_API_PROTOCOL=http
-REST_API_HOSTNAME=localhost
-REST_API_PORT=5123
-SOCKET_API_PROTOCOL=ws
-SOCKET_API_HOSTNAME=localhost
-SOCKET_API_PORT=5123
\ No newline at end of file
+PUBLIC_REST_API_PROTOCOL=http
+PUBLIC_REST_API_HOSTNAME=localhost
+PUBLIC_REST_API_PORT=5123
+PUBLIC_SOCKET_API_PROTOCOL=ws
+PUBLIC_SOCKET_API_HOSTNAME=localhost
+PUBLIC_SOCKET_API_PORT=5123
```

Comment:
Hvorfor hedder de public nu ?

Reply:
Det er en sveltekit ting, så hvis de er PUBLIC, så har svelte ikke noget imod at expose dem på client-side. Hvis de hedder SECRET så kan de kun køre på serverside.

Reply:
ah, okay tak for opklaring 👌